### ISSUE TITLE
400 for create-file

### ISSUE THREAD

**Issue by bbss** [wkok/openai-clojure#34]
I've just upgraded from 0.10.0 to 0.11.0 to give fine-tuning a spin (hopefully gpt-4 fine-tuning will be announced on monday!)

But when running:
```clojure
(wkok.openai-clojure.api/create-file
 {:purpose "fine-tune"
  :file (clojure.java.io/file "one.jsonl")}
 {:api-key openai-key})
```

I get a 400 for the options object.

```
ExceptionInfo
   status: 400
   {:request-time 876,
    :request
    {:user-info nil,
     :as :text,
     :headers
     {"Accept" "application/json",
      "Authorization"
      "Bearer 🐻 ",
      "content-type"
      "multipart/form-data; boundary=hatoBoundaryQTa_33CmKkdIY26ZhtH98PDN6JM1t0",
      "accept-encoding" "gzip, deflate"},
     :server-port nil,
     :url "https://api.openai.com/v1/files",
     :http-request
     #object[jdk.internal.net.http.HttpRequestImpl 0x166de34c "https://api.openai.com/v1/files POST"],
     :uri "/v1/files",
     :server-name "api.openai.com",
     :version :http-1.1,
     :query-string nil,
     :body
     #object[java.io.PipedInputStream 0x41bf8c3c "java.io.PipedInputStream@41bf8c3c"],
     :scheme :https,
     :request-method :post},
    :http-client
    #object[jdk.internal.net.http.HttpClientFacade 0x3b2f4320 "jdk.internal.net.http.HttpClientImpl@5913bcdb(9)"],
    :headers
    {"openai-organization" "user-me",
     "server" "cloudflare",
     "content-type" "application/json",
     "access-control-allow-origin" "*",
     "content-length" "177",
     "alt-svc" "h3=\":443\"; ma=86400",
     "openai-version" "2020-10-01",
     "strict-transport-security" "max-age=15724800; includeSubDomains",
     "openai-processing-ms" "77",
     "connection" "keep-alive",
     "cf-cache-status" "DYNAMIC",
     "cf-ray" "820d386008c96718-AMS",
     "date" "Sat, 04 Nov 2023 13:31:57 GMT",
     "x-request-id" "93f9bb55cb879c92f269b550f44fb1e7"},
    :status 400,
    :content-type :application/json,
    :uri "https://api.openai.com/v1/files",
    :content-type-params {},
    :version :http-1.1,
    :body
    "{\n  \"error\": {\n    \"message\": \"Additional properties are not allowed ('options' was unexpected)\",\n    \"type\": \"invalid_request_error\",\n    \"param\": null,\n    \"code\": null\n  }\n}\n"}
```

I hadn't tried this on 0.10.0 yet, so I don't think it's a regression. 

Any ideas?

**Comment by wkok** [comment]
Interesting, maybe OpenAI is more strict lately with request validation, but nonetheless, I fixed it so we do not ad an `options` key into the multipart fields.
Released [v0.11.1](https://github.com/wkok/openai-clojure/releases/tag/0.11.1)

**Comment by bbss** [comment]
Cheers! Thanks for the quick turnaround :)
